vnext/Scripts/OpenSSL.nuspec

```diff
@@ -7,20 +7,33 @@
     <authors>Microsoft</authors>
```

Comment:
is this right?

Reply:
Not sure, to be honest.

DevDiv used to publish a NuGet which had that field value.
If needed, we can update this on a subsequent change and generate a new version.